shared/js/background/dnr-session-rule-id.js

```diff
@@ -17,7 +22,7 @@ export async function setSessionRuleOffsetFromStorage () {
 }
 
 /**
- * Get the next unique session rule id to use when craeting session DNR rules
+ * Get the next unique session rule id to use when creating session DNR rules
```

Comment:
Nice, thanks for fixing some typos :1st_place_medal:

Reply:
👍